### PR TITLE
Move layer between water level & important POI labels

### PR DESCRIPTION
**Pull request by Sense**
This PR is linked to the new basic style update in [this other PR](https://github.com/QwantResearch/qwant-basic-gl-style/pull/69)

### CODE REVIEW

**Comment by amatissart** [comment]
It looks good, but works only for the route that is initially active. Other calls to `moveLayer` should be modified too : 

routes_road_names

---

SIde note: It could be useful to define the layerId (`"poi-level-street-furniture"` or another layer all routes should be drawn before) as a constant somewhere, for maintainability.

**Comment by Sense** [comment]
> It looks good, but works only for the route that is initially active. Other calls to `moveLayer` should be modified too :
> 
> routes_road_names
> 
> SIde note: It could be useful to define the layerId (`"poi-level-street-furniture"` or another layer all routes should be drawn before) as a constant somewhere, for maintainability.

I defined the `routes_layer` constants in the `config/constants.yml` file under the `map` section.

I also specified to move the direction layer below the `routes_layer` in `setMainRoute` func which is triggered when switching to an alternate possible route.

File: src/adapters/scene_direction.js [hunk 1]
```diff
@@ -65,15 +67,10 @@ export default class SceneDirection {
...
 
   setMainRoute(routeId) {
-    let mainRoute = null;
     this.routes.forEach((route) => {
-      if (route.id === routeId) {
-        mainRoute = route
-      }
       this.map.setFeatureState({source: `source_${route.id}`, id: 1}, {isActive: route.id === routeId})
     })
-    this.updateMarkers(mainRoute)
```
Comment (amatissart) [inline src/adapters/scene_direction.js]: The merge conflict has not been correctly fixed: these lines are part from a previous PR (#174) and should not be removed.

Comment (Sense) [reply src/adapters/scene_direction.js]: Thank you, just the restored the function as it previously was.

**Comment by Sense** [comment]
> It looks great.
> Could you also update the style version to apply your changes from [QwantResearch/qwant-basic-gl-style#69](https://github.com/QwantResearch/qwant-basic-gl-style/pull/69) ?

I updated the `package.json` and `package-lock.json` files in order to use the updated map style.
Thanks.

**Pull request merged**